packages/edit-site/src/store/index.js

```diff
@@ -23,8 +23,20 @@ export default function registerEditSiteStore( initialState ) {
 		initialState,
 	} );
 
+	const { showOnFront, pageOnFront } = initialState.home;
+	const initialPage = {
```

Comment:
Ultimately, it would be nice to be able to set this value externally from Gutenberg. It would also be nice to just pass in `initialRoute: '/'` (or even a page ID) and then edit site resolves everything else which is needed.

Reply:
> Ultimately, it would be nice to be able to set this value externally from Gutenberg.

Not sure I'm following, can you elaborate?

> It would also be nice to just pass in `initialRoute: '/'` (or even a page ID) and then edit site resolves everything else which is needed.

I think that's kind of the point of `findTemplate` -- pass a route, and get the associated template.

> (or even a page ID)

I think a route is preferable, since it can be used for non-page entities (categories etc)?

Reply:
> Not sure I'm following, can you elaborate?

1. deep linking into the site editor to access a specific page
2. from a different integration of the site editor, telling it the initial page/route to load

basically, just allowing you to modify the default page loaded by the editor

> I think that's kind of the point of findTemplate

`findTemplate` doesn't return the required page/post ID which would be needed to load the post content.

> I think a route is preferable

I tend to agree :) I was thinking of a situation where we know the post ID / page ID and want to tell the site editor to load that post/page